### PR TITLE
Implemented the generic modal

### PR DESCRIPTION
**Pull request by lakshayman**
Implemented generic modal on get intro button form on members profile page for devUser.

### CODE REVIEW

File: src/components/UI/modal/index.jsx [hunk 1]
```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from 'components/UI/modal/modal.module.scss';
-import Backdrop from 'components/UI/backdrop';
+import styles from '@components/UI/modal/modal.module.scss';
+import Backdrop from '@components/UI/backdrop';
 
-const Modal = ({ show, closeModal, children }) => (
+const Modal = ({ show, closeModal, modalChildren, style }) => (
```
Comment (sumitd94) [inline src/components/UI/modal/index.jsx:6]: why `modalChildren`? it's already a Modal component and we do not have to explicitly say the children as `modalChildren`. just `children` should work fine.

File: src/components/UI/modal/index.jsx [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+        ...style,
         transform: show ? 'translateY(0)' : 'translateY(-100vh)',
         opacity: show ? '1' : '0',
       }}
     >
       <span className={styles.crossIcon}>
         <i className="fa fa-times" aria-hidden="true" onClick={closeModal} />
       </span>
-      {children}
+      {modalChildren}
```
Comment (sumitd94) [inline src/components/UI/modal/index.jsx:20]: please change it to `children`

File: src/components/UI/modal/index.jsx (hunk 1 above)
Comment (lakshayman) [reply src/components/UI/modal/index.jsx]: done changed

File: src/components/UI/modal/index.jsx (hunk 2 above)
Comment (lakshayman) [reply src/components/UI/modal/index.jsx]: done

**Pull request closed without merging**